chains/sapphiretestnet/metadata.yaml

```
@@ -0,0 +1,19 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://nexus.oasis.io/v1/
+    family: other
+    name: Oasis Explorer
+    url: https://explorer.oasis.io/testnet/sapphire
```

add `blocks` information e.g.
```suggestion
    url: https://explorer.oasis.io/testnet/sapphire
blocks:
  confirmations: 3
  estimateBlockTime: 2
  reorgPeriod: 2
```

if it's on-demand block production, set confirmations to 1, reorgPeriod to 0, and use a rough estimate block time assuming constant load